src/discord-cluster-manager/submission.py

```diff
@@ -49,7 +49,8 @@ def prepare_submission(req: SubmissionRequest, lb_db: LeaderboardDB) -> Processe
         for g in req.gpus:
             if g not in task_gpus:
                 task_gpu_list = "".join([f" * {t}\n" for t in task_gpus])
-                raise KernelBotError(
+
+                raise ValueError(
```

Comment:
that's a message that we want to get through to the user, so it should be a KernelBotError.